### ISSUE TITLE
1.3.0 appears to break ST3 compatibility

### ISSUE THREAD

**Issue by juliangarnier** [sublimelsp/LSP-html#45]
Hi, I'm using ST 3211 (had to stick to ST3 for some reasons) and it looks like LSP-html is incompatible with ST3 since 1.3.0.
I installed LSP-html via package control, restarted Sublime and got this error in the console:

```
reloading plugin LSP-html.plugin
Traceback (most recent call last):
  File "/Applications/Sublime Text 3.app/Contents/MacOS/sublime_plugin.py", line 125, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.3/importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/Applications/Sublime Text 3.app/Contents/MacOS/sublime_plugin.py", line 1199, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "/Users/juliangarnier/Library/Application Support/Sublime Text 3/Installed Packages/LSP-html.sublime-package/plugin.py", line 2, in <module>
ImportError: cannot import name Session
```

Manually downloading 1.2.9 to my Packages folder fixed the issue.

I know that ST3 is probably not your priority, but wanted to surface this issue just in case you didn't noticed.

I also have the same problem with LSP-css (going to post an issue there as well).

Thanks.

**Comment by jfcherng** [comment]
Thanks. Fixed in [1.3.4](https://github.com/sublimelsp/LSP-html/releases/tag/1.3.4).

**Comment by juliangarnier** [comment]
I have a new error with 1.3.4, similar to lsp-css (https://github.com/sublimelsp/LSP-css/issues/46)

**Comment by jfcherng** [comment]
Closed since `[st3-1.4.0](https://github.com/sublimelsp/LSP-html/releases/tag/st3-1.4.0)` has been on Package Control.
